CODEOWNERS

```
@@ -707,7 +707,7 @@
 /scripts/nrf_provision/fast_pair/*.rst    @nrfconnect/ncs-si-bluebagel-doc
 /scripts/partition_manager/*.rst          @nrfconnect/ncs-aurora-doc
 /scripts/shell/ble_console/**/*.rst       @nrfconnect/ncs-doc-leads
-/scripts/west_commands/sbom/*.rst         @nrfconnect/ncs-si-muffin-doc
+/scripts/west_commands/sbom/*.rst         @nrfconnect/ncs-co-doc
```

Just to ensure we have a techwriter involved.
```suggestion
/scripts/west_commands/sbom/*.rst         @nrfconnect/ncs-co-doc @nrfconnect/ncs-doc-leads
```